backend/src/crash/main.py

```diff
@@ -194,15 +223,28 @@ async def websocket_endpoint(websocket: WebSocket):
     except WebSocketDisconnect:
 
         manager.disconnect(websocket)
-        new_player_list = list(current_players.values())
-        new_player_list.remove(current_players[websocket])
+        new_player_list = [player.name for player in current_players.values()]
```

Comment:
I understand this logic was already here prior to this PR, but I'm pretty sure this will not send out the full lobby to players in the following scenario:
```
A, B, C are in the lobby
round starts
-> D joins
game is played
<- C leaves
>> broadcast with lobby state set to [A, B] even though D should be there (we expect [A, B, D])
```
This is caused by this logic that uses `current_players` and not `game_handler.players`